.github/workflows/continous-integration.yml

```diff
@@ -45,7 +46,16 @@ jobs:
         with:
           python-version: ${{matrix.python-version}}
           architecture: x64
-
+      - name: Init Template
+        env:
+          python_version: ${{matrix.python-version}}
+        run: |
+          cd bin
+          echo $python_version
```

Comment:
I suppose this line is just for debug sake, am I wrong?

Reply:
yes. i will remove it straight away